cortx-monitorQuickstartGuide.md

```diff
@@ -137,23 +137,80 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
     -rw-r--r-- 1 root root   9592 Aug 18 07:27 cortx-libsspl_sec-method_pki-1.0.0-1_git8907300.el7.x86_64.rpm
     -rw-r--r-- 1 root root 101004 Aug 18 07:27 cortx-libsspl_sec-devel-1.0.0-1_git8907300.el7.x86_64.rpm
     ```
-
-3. Install CORTX-Monitor RPMs
+    
+5. Install CORTX-Monitor RPMs
 
    You'll need to create a repository and copy all the required RPMs to it. Run the commands:
 
     ```shell
-    
     $ mkdir MYRPMS
     $ cp -R /root/rpmbuild/RPMS/x86_64/cortx-libsspl_sec-* /root/rpmbuild/RPMS/noarch/cortx-sspl-* MYRPMS
-    $ find MYRPMS -name \*.rpm -print0 | sudo xargs -0 yum install -y
+    $ CORTX_MONITOR_BASE_URL="https://raw.githubusercontent.com/Seagate/cortx-monitor/main"
+    $ curl $CORTX_MONITOR_BASE_URL/low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy -o sspl_dev_deploy
+    $ chmod a+x sspl_dev_deploy
+    $ ./sspl_dev_deploy --cleanup
+    $ ./sspl_dev_deploy --prereq -L /root/MYRPMS
+    ```
+   
+   Create template file with the name `/root/template_values.1-node.txt`
+    ```
+    # 1-node config variable
+    TMPL_CLUSTER_ID=CC01
+    TMPL_NODE_ID=SN01
+    TMPL_RACK_ID=RC01
+    TMPL_SITE_ID=DC01
+    TMPL_MACHINE_ID=0449364d92b2ba3915fcd8416014cff7
+    TMPL_HOSTNAME=ssc-vm-4778.colo.seagate.com
+    TMPL_NODE_NAME=srvnode-1
+    TMPL_SERVER_NODE_TYPE=VM
+    TMPL_MGMT_INTERFACE=eth0
+    TMPL_MGMT_PUBLIC_FQDN=localhost
+    TMPL_DATA_PRIVATE_FQDN=localhost
+    TMPL_DATA_PRIVATE_INTERFACE=
+    TMPL_DATA_PUBLIC_FQDN=localhost
+    TMPL_DATA_PUBLIC_INTERFACE=
+    TMPL_BMC_IP=
+    TMPL_BMC_USER=
+    TMPL_BMC_SECRET=""
+
+    TMPL_ENCLOSURE_ID=enc_0449364d92b2ba3915fcd8416014cff7
+    TMPL_ENCLOSURE_NAME=enclosure-1
+    TMPL_ENCLOSURE_TYPE=VM
+    TMPL_PRIMARY_CONTROLLER_IP=127.0.0.1
+    TMPL_PRIMARY_CONTROLLER_PORT=28200
+    TMPL_SECONDARY_CONTROLLER_IP=127.0.0.1
+    TMPL_SECONDARY_CONTROLLER_PORT=28200
+    TMPL_CONTROLLER_USER=manage
+    TMPL_CONTROLLER_SECRET=""
+    TMPL_CONTROLLER_TYPE=Gallium
+    ```
+    Make sure to update TMPL_MACHINE_ID, TMPL_HOSTNAME, TMPL_ENCLOSURE_ID, TMPL_BMC_SECRET, TMPL_CONTROLLER_SECRET with actual values.
+    
+    To generate passwords you can use below steps.
+    ```
+    machine_id="xyz"
+    enclosure_id="enc_xyz"
+    bmc_pass='Secret'
+    encl_pass='Secret'
+
+    bmc_key=$(python3 -c 'from cortx.utils.security.cipher import Cipher; print(Cipher.generate_key('"'$machine_id'"', "server_node"))')
+    encl_key=$(python3 -c 'from cortx.utils.security.cipher import Cipher; print(Cipher.generate_key('"'$enclosure_id'"', "storage_enclosure"))')
+
+    #Encryption
```

Comment:
this shows CORTX encryption mechanism, which anyways provisioner would be covering in deployment guide, to enable community users to feed proper encrypted passwords to required config.
Still @johnbent  please confirm if this is fine.

Reply:
Is the secret key merely created by hashing publicly known data?  Shouldn't we include something random or a pass phrase from the user?

Reply:
Above code is just merely showing a way with which key generation can happen, where for simplicity we are feeding some known data from configuration. As part of Solution encryption logic, consumer is free to feed any input to generate keys.
Probably we should add that as a Note here !? Would be good to relate faster i guess.

@thavanathan can you please add that.

Reply:
@cdeshmukh added a note.